apis/v1alpha1/opentelemetrycollector_types.go

```diff
@@ -275,6 +276,11 @@ type OpenTelemetryCollectorSpec struct {
 	// object, which shall be mounted into the Collector Pods.
 	// Each ConfigMap will be added to the Collector's Deployments as a volume named `configmap-<configmap-name>`.
 	ConfigMaps []ConfigMapsSpec `json:"configmaps,omitempty"`
+	// UpdateStrategy represents the strategy the operator will take replacing existing DaemonSet pods with new pods
+	// https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/daemon-set-v1/#DaemonSetSpec
+	// This is only applicable to Daemonset mode.
```

Comment:
Does it make sense to validate this in the validation webhook?

Reply:
Good point! I'll add a validation to enable this field only for DaemonSet mode.

Reply:
Just added the validation you mentioned. Could you please have a look?